kotlin-source/src/test/kotlin/com/example/contract/IOUFlowTests.kt

```diff
@@ -144,12 +79,8 @@ class IOUFlowTests {
 
     @Test
     fun `recorded transaction has no inputs and a single output, the input IOU`() {
-        val inputState = IOUState(
-                IOU(1),
-                a.info.legalIdentity,
-                b.info.legalIdentity,
-                IOUContract())
-        val flow = ExampleFlow.Initiator(inputState, b.info.legalIdentity)
+        val iouValue = 1
```

Comment:
It's reused - once when kicking off the flow, and once when checking the output of the flow. Means that if you change the value in one place but not the other, it won't break.